services/identity-service/src/main.ts

```diff
@@ -0,0 +1,22 @@
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { AppModule } from './app.module';
+import { Logger } from '@nestjs/common';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  app.enableCors()
+  // setup swagger
+  const config = new DocumentBuilder()
+    .setTitle('Identity Service')
+    .setDescription('The Identity Service API description')
+    .setVersion('1.0')
```

Comment:
Move this to package.json and get it from there for easy versioning.

Reply:
done